### ISSUE TITLE
Operations not supporting uint8

### ISSUE THREAD

**Issue by fraxx001** [tensorflow/tflite-micro#630]
Hi,
I still have the problem same issue as posted in the [tf-issue](https://github.com/tensorflow/tensorflow/issues/47610). In my case the TF_ENSURE operation fails in softmax-common because the datatype is uint8. This is very annoying because all tensorflow-hub models are trained with uint8 and it is very tedious to convert them.
